web/app/css/styles.css

```diff
@@ -1,5 +1,5 @@
 /* material-ui uses Roboto */
-@import url("https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:100,300,400");
+@import url("https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:100,300,400|Roboto+Mono");
```

Comment:
Would it be possible to add `Roboto Mono` font to get a `terminal` aspect for the modal?

Reply:
@cynthia-sg, that sounds good to me. 👍